manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "Copenhagen",
   "author": "Zendesk",
-  "version": "4.3.0-beta.6",
+  "version": "4.3.0",
```

Comment:
Once merged on `beta`, a new release will be created and this will change to `4.4.0-beta.1`